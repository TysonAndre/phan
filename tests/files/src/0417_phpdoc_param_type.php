<?php

class Subclass extends BaseClass {
    /**
     * @param array|\Traversable $arg
     * @return stdClass
     */
    public function method($arg = null) {
        return new stdClass();
    }
}

class BaseClass {
    /**
     * @param array|\Traversable $arg
     * @return void
     */
    public function method($arg = null) {
    }
}
