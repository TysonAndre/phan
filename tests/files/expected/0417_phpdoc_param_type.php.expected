%s:8 PhanParamSignatureMismatch Declaration of function method(\Traversable|array $arg = null) : \stdClass should be compatible with function method(\Traversable|array $arg = null) : void defined in %s:18
